AdminWebsite/AdminWebsite/ClientApp/src/app/booking/search-email/search-email.component.ts

```diff
@@ -48,10 +49,22 @@ export class SearchEmailComponent implements OnInit, OnDestroy {
 
     @Output() emailChanged = new EventEmitter<string>();
 
-    constructor(private searchService: SearchService, private configService: ConfigService, private logger: Logger) {}
+    constructor(
+        private searchService: SearchService,
+        private configService: ConfigService,
+        private logger: Logger,
+        private featureFlagService: FeatureFlagService
+    ) {}
 
     ngOnInit() {
         this.email = this.initialValue;
+        this.featureFlagService
+            .getFeatureFlagByName('EJudFeature')
+            .pipe(first())
+            .subscribe(result => {
+                this.judiciaryRoles = result ? Constants.JudiciaryRoles : [];
```

Comment:
I am not sure how this has worked? are we setting judiciaryRoles property here or checking the value against result? If its a condition we should use triple equals here

Reply:
explained in the above comment